tests/pebble.rs

```diff
@@ -50,12 +50,10 @@ async fn http_01() -> Result<(), Box<dyn StdError>> {
 
     // Spawn a Pebble CA and a challenge response server.
     debug!("starting Pebble CA environment");
-    let mut pebble = Environment::new().await?;
```

Comment:
I feel like we're flip-flopping on this -- not sure if you had other reasons than the authorization method repetition to keep these separate?

Reply:
I don't remember now. In the current moment I agree it makes sense to consolidate :+1: